.github/ISSUE_TEMPLATE/questions-help-support.md

```diff
@@ -0,0 +1,21 @@
+---
+name: "\U00002753Questions/Help/Support"
+about: Please let us know if you have any questions.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+## Question
+
+Please write your question here.
+
+### Note to the questioner
```

Comment:
How about placing this section first as that order seems more logical?

Reply:
You can probably ignore this comment as the current layout looks good without the rearrangement (it might just make it confusing).

Reply:
Okay. I'll keep the current layout.